repository/movie/src/main/java/com/repository/movie/mapper/MovieMapper.kt

```diff
@@ -114,9 +117,13 @@ fun MovieLogoDto.toEntity(id: Int): Image {
 
 fun MovieReviewDto.toEntity(): Review {
     return Review(
+        createdAt = try {
+            LocalDate.parse(this.createdAt.orEmpty())
```

Comment:
you can edit this to be like this 
```suggestion
            LocalDate.parse(this.createdAt ?: "9999,1,1")
```

Reply:
Sometimes the API returns emty string this will not handle it